static/app/components/teamRoleSelect.tsx

```diff
@@ -30,30 +29,21 @@ function TeamRoleSelect({
   const {orgRoleList, teamRoleList, features} = organization;
   const hasTeamRoles = features.includes('team-roles');
 
-  // Determine the org-role, including if the current team has an org role
-  // and adding the user to the current team changes their minimum team-role
-  const possibleOrgRoles = [member.orgRole];
-  if (member.groupOrgRoles && member.groupOrgRoles.length > 0) {
-    possibleOrgRoles.push(member.groupOrgRoles[0].role.id);
-  }
-  if (team.orgRole) {
-    possibleOrgRoles.push(team.orgRole);
-  }
-  const effectiveOrgRole = getEffectiveOrgRole(possibleOrgRoles, orgRoleList);
+  const memberOrgRole = orgRoleList.find(r => r.id === member.orgRole);
```

Comment:
does `memberOrgRole` here refer to the unreleased org role, and if so will we eventually delete all references to it?

Reply:
it is the member's org role but as an `OrgRole` type of object. `member.orgRole` is a string